doc/rtd/applications.rst

```diff
@@ -484,7 +484,7 @@ Example
 .. code-block:: bash
 
     mpiexec -np 10 ./bin/teca_tc_trajectory_scalars   \
-        --texture ../../TECA_data/earthmap4k.png      \
+        --texture ../../TECA_assets/earthmap4k.png      \
```

Comment:
we no longer need to pass on the command line, the default value can be obtained using the new api you defined in teca_py_config.i

Reply:
I don't think we should do that because this is given as a command line argument to the `--texture` option in the `teca_tc_trajectory_scalars` app. We can modify it to `/path/to/background_image.png` if that makes sense. Please let me know if I am missing something.

Reply:
You might be missing that the point of deploying the assets with TECA is that we no longer need to pass these on the command line. It's fine to leave the command line options but make them optional, with the default value pointing to the assets location. Same deal as with the pytorch model.